frontend/public/components/configmap-and-secret-data.jsx

```diff
@@ -1,14 +1,36 @@
 import * as React from 'react';
 import * as PropTypes from 'prop-types';
-import { Heading } from './utils';
+import { Heading, CopyToClipboard } from './utils';
 
-export const ConfigMapData = ({data}) => {
+const MaskedData = () => {
```

Comment:
I think my original comment got collapsed here, but this can just be

```jsx
const MaskedData = <React.Fragment>
  <span className="sr-only">Value hidden</span>
  <span aria-hidden="true">&bull;&bull;&bull;&bull;&bull;</span>
</React.Fragment>;
```